mmv1/products/bigquerydatapolicy/api.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Product
+name: BigqueryDatapolicy
+display_name: BigQuery Data Policy
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://bigquerydatapolicy.googleapis.com/v1beta1/
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: BigQuery Data Policy API
+    url: https://console.cloud.google.com/apis/library/bigquerydatapolicy.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: "DataPolicy"
+    min_version: beta
+    base_url: projects/{{project}}/locations/{{location}}/dataPolicies
+    create_url: projects/{{project}}/locations/{{location}}/dataPolicies
+    self_link: projects/{{project}}/locations/{{location}}/dataPolicies/{{data_policy_id}}
+    update_verb: :PATCH
+    update_mask: true
```

Comment:
Can you include a handwritten test that uses the update functionality?

Reply:
added